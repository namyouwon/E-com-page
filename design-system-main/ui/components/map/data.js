// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license

export const data = [
  {
    title: 'Worldwide Corporate Headquarters',
    address: 'The Landmark @ One Market, San Francisco, CA'
  },
  {
    title: 'salesforce.com inc Atlanta',
    address: '950 East Paces Ferry Road NE, Atlanta, GA'
  },
  {
    title: 'salesforce.com inc Bellevue',
    address: '929 108th Ave NE, Bellevue, WA'
  },
  {
    title: 'salesforce.com inc Boston',
    address: '500 Boylston Street 19th Floor, Boston, MA'
  },
  {
    title: 'salesforce.com inc Chicago',
    address: '111 West Illinois Street, Chicago, IL'
  },
  {
    title: 'salesforce.com inc Herndon',
    address: '2550 Wasser Terrace, Herndon, VA'
  },
  {
    title: 'salesforce.com inc Hillsboro',
    address: '2035 NE Cornelius Pass Road, Hillsboro, OR'
  },
  {
    title: 'salesforce.com inc Indy',
    address: '111 Monument Circle, Indianapolis, IN'
  },
  {
    title: 'salesforce.com inc Irvine',
    address: '300 Spectrum Center Drive, Irvine, CA'
  }
];
