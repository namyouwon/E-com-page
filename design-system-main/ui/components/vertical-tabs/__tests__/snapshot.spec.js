// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license
/* eslint-env jest */

import React from 'react';
import createHelpers from '../../../../jest.helpers';
import VerticalTabs from '../';
import * as Snapshot from '../base/snapshots.data';
const { matchesMarkup } = createHelpers(__dirname);

describe('Vertical Tabs', () => {
  it('renders base vertical tab items', () =>
    matchesMarkup(<VerticalTabs tabs={Snapshot.exampleTabs} />));
  it('renders base vertical tab items with icons', () =>
    matchesMarkup(<VerticalTabs tabs={Snapshot.exampleTabsWithIcons} />));
  it('renders base vertical tab items with badges', () =>
    matchesMarkup(<VerticalTabs tabs={Snapshot.exampleTabsWithBadges} />));
});
