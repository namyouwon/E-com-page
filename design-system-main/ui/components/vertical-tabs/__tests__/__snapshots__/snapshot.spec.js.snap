// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Vertical Tabs renders base vertical tab items 1`] = `
<div
  className="slds-vertical-tabs"
>
  <ul
    aria-orientation="vertical"
    className="slds-vertical-tabs__nav"
    role="tablist"
  >
    <li
      className="slds-vertical-tabs__nav-item slds-is-active"
      role="presentation"
      title="Tab 1"
    >
      <a
        aria-controls="slds-vertical-tabs-0"
        aria-selected="true"
        className="slds-vertical-tabs__link"
        href="#"
        id="slds-vertical-tabs-0__nav"
        onClick={[Function]}
        role="tab"
        tabIndex={0}
      >
        <span
          className="slds-vertical-tabs__left-icon"
        />
        <span
          className="slds-truncate"
          title="Tab 1"
        >
          Tab 1
        </span>
        <span
          className="slds-vertical-tabs__right-icon"
        />
      </a>
    </li>
    <li
      className="slds-vertical-tabs__nav-item"
      role="presentation"
      title="Tab 2"
    >
      <a
        aria-controls="slds-vertical-tabs-1"
        aria-selected="false"
        className="slds-vertical-tabs__link"
        href="#"
        id="slds-vertical-tabs-1__nav"
        onClick={[Function]}
        role="tab"
        tabIndex={-1}
      >
        <span
          className="slds-vertical-tabs__left-icon"
        />
        <span
          className="slds-truncate"
          title="Tab 2"
        >
          Tab 2
        </span>
        <span
          className="slds-vertical-tabs__right-icon"
        />
      </a>
    </li>
    <li
      className="slds-vertical-tabs__nav-item"
      role="presentation"
      title="Tab 3 has a really long label and can wrap or truncate"
    >
      <a
        aria-controls="slds-vertical-tabs-2"
        aria-selected="false"
        className="slds-vertical-tabs__link"
        href="#"
        id="slds-vertical-tabs-2__nav"
        onClick={[Function]}
        role="tab"
        tabIndex={-1}
      >
        <span
          className="slds-vertical-tabs__left-icon"
        />
        <span
          className="slds-truncate"
          title="Tab 3 has a really long label and can wrap or truncate"
        >
          Tab 3 has a really long label and can wrap or truncate
        </span>
        <span
          className="slds-vertical-tabs__right-icon"
        />
      </a>
    </li>
  </ul>
  <div
    aria-labelledby="slds-vertical-tabs-0__nav"
    className="slds-vertical-tabs__content slds-show"
    id="slds-vertical-tabs-0"
    role="tabpanel"
  >
    <div
      className="slds-text-longform"
    >
      <h3
        className="slds-text-heading_medium"
      >
        Tab Title
      </h3>
      <p>
        Content for Tab 1
      </p>
      <p>
        Lorem ipsum dolor...
      </p>
      <p>
        Lorem ipsum dolor...
      </p>
    </div>
  </div>
  <div
    aria-labelledby="slds-vertical-tabs-1__nav"
    className="slds-vertical-tabs__content slds-hide"
    id="slds-vertical-tabs-1"
    role="tabpanel"
  >
    <div
      className="slds-text-longform"
    >
      <h3
        className="slds-text-heading_medium"
      >
        Tab Title
      </h3>
      <p>
        Content for Tab 1
      </p>
      <p>
        Lorem ipsum dolor...
      </p>
      <p>
        Lorem ipsum dolor...
      </p>
    </div>
  </div>
  <div
    aria-labelledby="slds-vertical-tabs-2__nav"
    className="slds-vertical-tabs__content slds-hide"
    id="slds-vertical-tabs-2"
    role="tabpanel"
  >
    <div
      className="slds-text-longform"
    >
      <h3
        className="slds-text-heading_medium"
      >
        Tab Title
      </h3>
      <p>
        Content for Tab 1
      </p>
      <p>
        Lorem ipsum dolor...
      </p>
      <p>
        Lorem ipsum dolor...
      </p>
    </div>
  </div>
</div>
`;

exports[`Vertical Tabs renders base vertical tab items with badges 1`] = `
<div
  className="slds-vertical-tabs"
>
  <ul
    aria-orientation="vertical"
    className="slds-vertical-tabs__nav"
    role="tablist"
  >
    <li
      className="slds-vertical-tabs__nav-item slds-is-active"
      role="presentation"
      title="Opportunities"
    >
      <a
        aria-controls="slds-vertical-tabs-0"
        aria-selected="true"
        className="slds-vertical-tabs__link"
        href="#"
        id="slds-vertical-tabs-0__nav"
        onClick={[Function]}
        role="tab"
        tabIndex={0}
      >
        <span
          className="slds-vertical-tabs__left-icon"
        >
          <span
            className="slds-icon_container slds-icon-standard-opportunity"
            title="Description of icon when needed"
          >
            <svg
              aria-hidden={true}
              className="slds-icon slds-icon_small"
            >
              <use
                xlinkHref="/assets/icons/standard-sprite/svg/symbols.svg#opportunity"
              />
            </svg>
          </span>
        </span>
        <span
          className="slds-truncate"
          title="Opportunities"
        >
          Opportunities
        </span>
        <span
          className="slds-vertical-tabs__right-icon"
        >
          <span
            className="slds-badge slds-badge_inverse"
          >
            New
          </span>
        </span>
      </a>
    </li>
    <li
      className="slds-vertical-tabs__nav-item"
      role="presentation"
      title="Cases"
    >
      <a
        aria-controls="slds-vertical-tabs-1"
        aria-selected="false"
        className="slds-vertical-tabs__link"
        href="#"
        id="slds-vertical-tabs-1__nav"
        onClick={[Function]}
        role="tab"
        tabIndex={-1}
      >
        <span
          className="slds-vertical-tabs__left-icon"
        >
          <span
            className="slds-icon_container slds-icon-standard-case"
            title="Description of icon when needed"
          >
            <svg
              aria-hidden={true}
              className="slds-icon slds-icon_small"
            >
              <use
                xlinkHref="/assets/icons/standard-sprite/svg/symbols.svg#case"
              />
            </svg>
          </span>
        </span>
        <span
          className="slds-truncate"
          title="Cases"
        >
          Cases
        </span>
        <span
          className="slds-vertical-tabs__right-icon"
        >
          <span
            className="slds-badge slds-badge_inverse"
          >
            New
          </span>
        </span>
      </a>
    </li>
    <li
      className="slds-vertical-tabs__nav-item"
      role="presentation"
      title="Accounts"
    >
      <a
        aria-controls="slds-vertical-tabs-2"
        aria-selected="false"
        className="slds-vertical-tabs__link"
        href="#"
        id="slds-vertical-tabs-2__nav"
        onClick={[Function]}
        role="tab"
        tabIndex={-1}
      >
        <span
          className="slds-vertical-tabs__left-icon"
        >
          <span
            className="slds-icon_container slds-icon-standard-account"
            title="Description of icon when needed"
          >
            <svg
              aria-hidden={true}
              className="slds-icon slds-icon_small"
            >
              <use
                xlinkHref="/assets/icons/standard-sprite/svg/symbols.svg#account"
              />
            </svg>
          </span>
        </span>
        <span
          className="slds-truncate"
          title="Accounts"
        >
          Accounts
        </span>
        <span
          className="slds-vertical-tabs__right-icon"
        />
      </a>
    </li>
  </ul>
  <div
    aria-labelledby="slds-vertical-tabs-0__nav"
    className="slds-vertical-tabs__content slds-show"
    id="slds-vertical-tabs-0"
    role="tabpanel"
  >
    <div
      className="slds-text-longform"
    >
      <h3
        className="slds-text-heading_medium"
      >
        Tab Title
      </h3>
      <p>
        Content for Tab 1
      </p>
      <p>
        Lorem ipsum dolor...
      </p>
      <p>
        Lorem ipsum dolor...
      </p>
    </div>
  </div>
  <div
    aria-labelledby="slds-vertical-tabs-1__nav"
    className="slds-vertical-tabs__content slds-hide"
    id="slds-vertical-tabs-1"
    role="tabpanel"
  >
    <div
      className="slds-text-longform"
    >
      <h3
        className="slds-text-heading_medium"
      >
        Tab Title
      </h3>
      <p>
        Content for Tab 1
      </p>
      <p>
        Lorem ipsum dolor...
      </p>
      <p>
        Lorem ipsum dolor...
      </p>
    </div>
  </div>
  <div
    aria-labelledby="slds-vertical-tabs-2__nav"
    className="slds-vertical-tabs__content slds-hide"
    id="slds-vertical-tabs-2"
    role="tabpanel"
  >
    <div
      className="slds-text-longform"
    >
      <h3
        className="slds-text-heading_medium"
      >
        Tab Title
      </h3>
      <p>
        Content for Tab 1
      </p>
      <p>
        Lorem ipsum dolor...
      </p>
      <p>
        Lorem ipsum dolor...
      </p>
    </div>
  </div>
</div>
`;

exports[`Vertical Tabs renders base vertical tab items with icons 1`] = `
<div
  className="slds-vertical-tabs"
>
  <ul
    aria-orientation="vertical"
    className="slds-vertical-tabs__nav"
    role="tablist"
  >
    <li
      className="slds-vertical-tabs__nav-item slds-is-active"
      role="presentation"
      title="Opportunities"
    >
      <a
        aria-controls="slds-vertical-tabs-0"
        aria-selected="true"
        className="slds-vertical-tabs__link"
        href="#"
        id="slds-vertical-tabs-0__nav"
        onClick={[Function]}
        role="tab"
        tabIndex={0}
      >
        <span
          className="slds-vertical-tabs__left-icon"
        >
          <span
            className="slds-icon_container slds-icon-standard-opportunity"
            title="Description of icon when needed"
          >
            <svg
              aria-hidden={true}
              className="slds-icon slds-icon_small"
            >
              <use
                xlinkHref="/assets/icons/standard-sprite/svg/symbols.svg#opportunity"
              />
            </svg>
          </span>
        </span>
        <span
          className="slds-truncate"
          title="Opportunities"
        >
          Opportunities
        </span>
        <span
          className="slds-vertical-tabs__right-icon"
        />
      </a>
    </li>
    <li
      className="slds-vertical-tabs__nav-item"
      role="presentation"
      title="Cases"
    >
      <a
        aria-controls="slds-vertical-tabs-1"
        aria-selected="false"
        className="slds-vertical-tabs__link"
        href="#"
        id="slds-vertical-tabs-1__nav"
        onClick={[Function]}
        role="tab"
        tabIndex={-1}
      >
        <span
          className="slds-vertical-tabs__left-icon"
        >
          <span
            className="slds-icon_container slds-icon-standard-case"
            title="Description of icon when needed"
          >
            <svg
              aria-hidden={true}
              className="slds-icon slds-icon_small"
            >
              <use
                xlinkHref="/assets/icons/standard-sprite/svg/symbols.svg#case"
              />
            </svg>
          </span>
        </span>
        <span
          className="slds-truncate"
          title="Cases"
        >
          Cases
        </span>
        <span
          className="slds-vertical-tabs__right-icon"
        >
          <span
            className="slds-icon_container slds-icon-utility-error"
            title="This tab has an error"
          >
            <svg
              aria-hidden={true}
              className="slds-icon slds-icon_x-small slds-icon-text-error"
            >
              <use
                xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#error"
              />
            </svg>
            <span
              className="slds-assistive-text"
            >
              This tab has an error
            </span>
          </span>
        </span>
      </a>
    </li>
    <li
      className="slds-vertical-tabs__nav-item"
      role="presentation"
      title="Accounts"
    >
      <a
        aria-controls="slds-vertical-tabs-2"
        aria-selected="false"
        className="slds-vertical-tabs__link"
        href="#"
        id="slds-vertical-tabs-2__nav"
        onClick={[Function]}
        role="tab"
        tabIndex={-1}
      >
        <span
          className="slds-vertical-tabs__left-icon"
        >
          <span
            className="slds-icon_container slds-icon-standard-account"
            title="Description of icon when needed"
          >
            <svg
              aria-hidden={true}
              className="slds-icon slds-icon_small"
            >
              <use
                xlinkHref="/assets/icons/standard-sprite/svg/symbols.svg#account"
              />
            </svg>
          </span>
        </span>
        <span
          className="slds-truncate"
          title="Accounts"
        >
          Accounts
        </span>
        <span
          className="slds-vertical-tabs__right-icon"
        />
      </a>
    </li>
  </ul>
  <div
    aria-labelledby="slds-vertical-tabs-0__nav"
    className="slds-vertical-tabs__content slds-show"
    id="slds-vertical-tabs-0"
    role="tabpanel"
  >
    <div
      className="slds-text-longform"
    >
      <h3
        className="slds-text-heading_medium"
      >
        Tab Title
      </h3>
      <p>
        Content for Tab 1
      </p>
      <p>
        Lorem ipsum dolor...
      </p>
      <p>
        Lorem ipsum dolor...
      </p>
    </div>
  </div>
  <div
    aria-labelledby="slds-vertical-tabs-1__nav"
    className="slds-vertical-tabs__content slds-hide"
    id="slds-vertical-tabs-1"
    role="tabpanel"
  >
    <div
      className="slds-text-longform"
    >
      <h3
        className="slds-text-heading_medium"
      >
        Tab Title
      </h3>
      <p>
        Content for Tab 1
      </p>
      <p>
        Lorem ipsum dolor...
      </p>
      <p>
        Lorem ipsum dolor...
      </p>
    </div>
  </div>
  <div
    aria-labelledby="slds-vertical-tabs-2__nav"
    className="slds-vertical-tabs__content slds-hide"
    id="slds-vertical-tabs-2"
    role="tabpanel"
  >
    <div
      className="slds-text-longform"
    >
      <h3
        className="slds-text-heading_medium"
      >
        Tab Title
      </h3>
      <p>
        Content for Tab 1
      </p>
      <p>
        Lorem ipsum dolor...
      </p>
      <p>
        Lorem ipsum dolor...
      </p>
    </div>
  </div>
</div>
`;
